### PR TITLE
Switch from Debian to Alpine

### LINKED ISSUE #22: Base magic-wormhole image on Alpine rather than Debian
Following #21, we've reduced the size of the image to 190MB.
But we might even reach 120MB if we we decide to switch from `slim-bullseye` to `alpine3.17`!

Is this worth it?

### PR DESCRIPTION
**Pull request by btlogy**
Closes #22

### CODE REVIEW

File: wormhole/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,12 @@
-FROM pypy:3.9-7.3.9-bullseye
-
-COPY Pipfile /app/Pipfile
-COPY Pipfile.lock /app/Pipfile.lock
-
-RUN apt update
-RUN apt install rustc -y
+# Pull official base image from DockerHub
+FROM python:3.9.16-alpine3.17
```
Comment (hacklschorsch) [inline wormhole/Dockerfile:2]: Or maybe just `3.9-alpine`? (see [my other comment](https://github.com/LeastAuthority/magic-wormhole-docker/pull/21#discussion_r1187518065))

Comment (btlogy) [reply wormhole/Dockerfile:2]: See my other comment too: I'd rather making it explicit, since we need to merge and rebuild for minor update anyway.

Comment (btlogy) [reply wormhole/Dockerfile:2]: More on this in #29

**Comment by btlogy** [comment]
At the end, since we can not switch to Alpine for the relay and/or the mailbox, this pull_request has less value...
Maybe better work on Nix...

I'm closing for now. Feel free to re-open.

**Pull request closed without merging**